src/tv2-common/actions/actionTypes.ts

```diff
@@ -48,6 +48,10 @@ export interface ActionSelectJingle extends ActionBase {
 	clip: string
 }
 
+export interface ActionSelectRobotPreset extends ActionBase {
+	type: AdlibActionType.SELECT_ROBOT_PRESET
```

Comment:
_Select_ is usually indicating an AdLib Action that puts something in preview (does something affecting the next partInstance). I would prefer if the name of this interface and the action type used a verb that makes it more obvious that this happens immediately. I think _recall_ works fine, even though we're already using it in a slightly different context for the DVEs and Lives.

Reply:
Hmm, I wasn't aware of that that was how `SELECT` was used. Personally I think it would make sense to then rename `SELECT` to something that indicates it's being previewed, but I think that's a too large refactor for this PR.
I will update the name

Reply:
I've decided to call it `CALL_ROBOT_PRESET` as 'recall' indicate that we already called it atleast once which is most likely not the case

Reply:
>indicate that we already called it atleast once

That's not what is generally understood as _recalling a preset_, but _call_ works too.